.circle/deployment

```diff
@@ -18,6 +18,9 @@ then
   exit 1
 fi
 
+# make `gh` available for github API calls
+~/ci/.circle/install_gh
```

Comment:
This could be installed during the `dependencies` phase instead. The only gotcha is that it requires updating `.circleci/config.yml` for all packs (Why I'm doing now anyway, so that would not be a big deal).